Simple-LDAP-Login.php

```diff
@@ -21,8 +21,8 @@ class SimpleLDAPLogin {
     // openssl constants
     private static $openssl_method = "AES-256-CBC";
 
-    private static function get_openssl_pass() {
-        return gethostname();
```

Comment:
Wouldn't this break installations that don't use the local machine for LDAP? My `base_dn` is configured to point at our domain controller, and wordpress installations running on various servers point to the same host.